config

```diff
@@ -504,16 +504,6 @@ case "$GUESSOS" in
 	    OUT="darwin-i386-cc"
 	fi ;;
   x86_64-apple-darwin*)
-	if [ -z "$KERNEL_BITS" ]; then
-	    echo "WARNING! If you wish to build 32-bit library, then you have to"
-	    echo "         invoke 'KERNEL_BITS=32 $THERE/config $options'."
-	    if [ "$DRYRUN" = "false" -a -t 1 ]; then
-	      echo "         You have about 5 seconds to press Ctrl-C to abort."
-	      # The stty technique used elsewhere doesn't work on
-	      # MacOS. At least, right now on this Mac.
-	      sleep 5
-	    fi
-	fi
```

Comment:
This should probably be moved to it's own commit, since it seems unrelated to the other changes.